ovh/resource_dedicated_server_install_task.go

```diff
@@ -105,17 +105,25 @@ func resourceDedicatedServerInstallTask() *schema.Resource {
 							ForceNew:    true,
 							Description: "",
 						},
-						"ssh_key_name": {
+					},
+				},
+			},
+			"user_metadata": {
+				Type:     schema.TypeList,
+				Optional: true,
+				ForceNew: true,
+				MaxItems: 128,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"key": {
 							Type:        schema.TypeString,
```

Comment:
no, this list can be changed depending on the 'OS input questions' we are setting when we create an OS template

Reply:
ok, will you have a doc somewhere with examples / how to retrieve the available parameters by OS ? If we could link something in the provider doc it would be great

Reply:
https://help.ovhcloud.com/csm/en-dedicated-servers-api-os-installation?id=kb_article_view&sysparm_article=KB0061951#os-questions